news/772.bugfix

```diff
@@ -0,0 +1 @@
+Fix a packaging issue (missing sdist dependency)
```

Comment:
OmegaConf got a maintenance issue type? (file suffix)

Reply:
Nope, the suffixes defined in `pyproject.toml` are:
- feature
- bugfix
- api_change
- docs
- misc

Reply:
Want me to add maintenance?

Reply:
I think we can use misc for this.

Reply:
Done in https://github.com/omry/omegaconf/pull/776/commits/2f367b45e9248b5c9d00808df0a55a4b37eaff5a.